exercises/microwave/.meta/solutions/microwave.rb

```diff
@@ -0,0 +1,22 @@
+class Microwave
+  attr_reader :buttons_pressed, :minutes, :seconds
+
+  def initialize(input)
+    @buttons_pressed = input.join.to_i
+
+    base = buttons_pressed > 99 ? 100 : 60
```

Comment:
I'm not sure what you mean here. Isn't 99 the cutoff for seconds and then 100 would mean 1 minute?

Reply:
Yes, that is true... but combined, the microwave seems to accept it as a full input, and I would end up running my microwave for 100 minutes and 39 seconds.  It is smart enough to know that 99 seconds means 99 seconds... and so counts down from there, and then displays with modulo 60.

I am not sure that  > 99 being 100 seconds, because that means that 101 would be 100, instead of 1 minute 1 second..  To be sure, I am only reading this statement (without appreciating the "base" variable name), in the narrowest context and so perhaps it is intended that it is impossible to enter more than 99 in any of the two given sections.  Which also means that greater than 99 would be impossible in that case.